### PR TITLE
miniAOD customization sequence for 80X input

### PR DESCRIPTION
**Pull request by mverzett**
Added a customization file and function to allow to run the miniAOD sequence in 91X from 80X AOD input files.

@arizzi @gpetruc @ferencek @ronchese are you OK with this solution?

### CODE REVIEW

File: PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_80Xto91XreMiniAOD_BTV.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_80Xto91XreMiniAOD_BTV.py]: I think that the "current release" label should be removed from the file name.
Unless the number of customizations becomes huge, I'd rather keep it all in one file.

How about customizeMiniAOD_80XLegacyInput.py

Comment (mverzett) [inline PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_80Xto91XreMiniAOD_BTV.py]: Sure, no problem.

**Comment by arizzi** [comment]
Is this rerunning all tagger or only deepCSV?

**Comment by mverzett** [comment]
@arizzi , good catch! I am still getting the hang of the new Task feature, so I included more than was actually needed. I made sure that now it only runs deepCSV.

**Comment by arizzi** [comment]
@slava77 it seems that PPD is not very happy with top level customize. Would it make sense to have an "Era" just for this (something like Legacy2016) ?

**Comment by slava77** [comment]
@arizzi 
can the PPD comment in this thread or can I be added to that discussion.

miniAOD in its completeness is a customization, which doesn't play well with eras.

**Comment by davidlange6** [comment]
Closing PR. CMSSW_9_1_X is for muon TDR upgrade changes only (CMSSW_9_2_X for data taking)

**Comment by arizzi** [comment]
should @mverzett reopen against master branch?

**Comment by davidlange6** [comment]
everything goes to the master branch.. so yes.

**Comment by mverzett** [comment]
From what IB/anything I should start to make the new PR. If I simply take my branch and I try to push it to master I see a **lot** of unwanted commits

**Comment by arizzi** [comment]
I guess you can cherry-pick or rebase your commit(s) on any recent IB

**Comment by arizzi** [comment]
@mverzett  please anyhow do not delete this branch as @ronchese  may use it for his tests

**Comment by perrotta** [comment]
This PR is so simple (just one file added) that it will proabably be faster to you just add that file to the latest 92X IB, make a commit, and then a new PR out of it

**Comment by mverzett** [comment]
OK, thanks, I will do. For the record: master is then in sync with whichever latest and greatest IB. Do I get it right?

**Comment by davidlange6** [comment]
each 92x IB is created from the master at the moment it is built. yes.

**Comment by cerminar** [comment]
Hello @slava77 @arizzi , about the customize issue: my point is only that customize functions are quite error prone and we tend to forget quickly about their need.
Not sure there is another solution. Maybe one could exploit the PAT:sequencename functionality of ConfigBuilder: https://github.com/cms-sw/cmssw/blob/master/Configuration/Applications/python/ConfigBuilder.py#L1652 ?

Don't take me wrong, no intention to be difficult just trying to understand what would make things easier on the long term.

**Comment by davidlange6** [comment]
> On May 23, 2017, at 11:05 AM, cerminar <notifications@github.com> wrote:
> 
> Hello @slava77 @arizzi , about the customize issue: my point is only that customize functions are quite error prone and we tend to forget quickly about their need.
> Not sure there is another solution. Maybe one could exploit the PAT:sequencename functionality of ConfigBuilder: https://github.com/cms-sw/cmssw/blob/master/Configuration/Applications/python/ConfigBuilder.py#L1652 ?

this would be much prefered to either customize or era (mis)usage..

**Comment by slava77** [comment]
On 5/23/17 2:10 AM, David Lange wrote:
> 
>> On May 23, 2017, at 11:05 AM, cerminar <notifications@github.com> wrote:
>> 
>> Hello @slava77 @arizzi , about the customize issue: my point is only that customize functions are quite error prone and we tend to forget quickly about their need.
>> Not sure there is another solution. Maybe one could exploit the PAT:sequencename functionality of ConfigBuilder: https://github.com/cms-sw/cmssw/blob/master/Configuration/Applications/python/ConfigBuilder.py#L1652 ?
> 
> this would be much prefered to either customize or era (mis)usage..

I'm not sure what exactly is proposed here with "PAT:sequencename".
It clearly sounds like a modification is needed for prepare_PAT method
in the ConfigBuilder,
however I do not clearly see what it should be (sequences do not exist
in PAT,
so the "sequencename" will probably have to be interpreted as a
customization call to a method named "sequencename" which apparently
will allow only one such customization).

David,
please specify in more details what should be done or maybe implement
this, since you prefer this option.

**Pull request closed without merging**